spec/features/admin/index_spec.rb

```diff
@@ -8,12 +8,13 @@
       expect(page).to have_content("Admin Dashboard")
     end
     
-    it "should display links to admin/merchants and admin/invoices admin/dashboard" do
+    it "should display links to admin/merchants, admin/invoices, and admin/dashboard" do
       visit admin_path
-      save_and_open_page
+      
       expect(page).to have_link("Dashboard", :href => admin_path)
       expect(page).to have_link("Merchants", :href => admin_merchants_path)
       expect(page).to have_link("Invoices", :href => admin_invoices_path)
```

Comment:
resources for the win! also, like that we're testing the path too instead of just the title of the link 🏆